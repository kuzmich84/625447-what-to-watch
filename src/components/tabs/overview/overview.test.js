import React from "react";
import renderer from "react-test-renderer";
import Overview from "./overview";
import mockFilmList from "../../../mocks/films";


test(`Should Overview component render correctly`, () => {
  const tree = renderer
    .create(<Overview film={mockFilmList[3]}/>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
