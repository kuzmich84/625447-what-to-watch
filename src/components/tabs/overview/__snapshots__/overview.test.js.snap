// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Overview component render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      7.2
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      After Queen Elizabeth I commands him not to grow old, a young nobleman struggles with love and his place in the world.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Sally Potter
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Tilda Swinto, Billy Zane, Quentin Crisp
         and other
      </strong>
    </p>
  </div>,
]
`;
